### ISSUE TITLE
MaskRCNN to TFLite 

### ISSUE THREAD

**Issue by sauravsolanki** [matterport/Mask_RCNN#2611]
There are many issues open that talk about conversion of Model from tensorflow to TFLite but none has straight answer to it. If anyone has able to do it by custom ops, please post it here. It would be really great if you can share the code. Any kind of knowledge sharing would be great. I tried converting the model but I got error(Segmentation Fault) while inference.

If nobody has done it, I would like to contribute to addition of TF Lite feature. It would be great help to our community.

**Comment by nataliameira** [comment]
I would like to know too. You managed to convert. The model didn't work?

**Comment by sauravsolanki** [comment]
Yes @nataliameira but it was not successful. Noisy outputs at the end.

**Comment by TranATT** [comment]
@sauravsolanki I converted it to Tflite also and I also got the same error like you during inference :(.
